README.md

```diff
@@ -1,79 +1,53 @@
-# <h1 align="center"> Balancer Tools </h1>
+# bleu's Balancer Tools
 
 The Balancer Tools repository serves as a hub for open-source projects aimed at empowering Balancer users with additional functionalities, analytics, and ease of use. Here, you will find a variety of tools and utilities that can be used to augment your experience as a Balancer user, whether you're a liquidity provider, trader, Maxi or developer.
 
-## Avaliable tools
-
-### [`Pool Metadata`](./packages/balancer-tools/src/app/metadata/)
-
-Balancer Pool Metadata is that enables pool owners to add metadata to their pools on the Balancer platform. This metadata can include information such as pool name, symbol, description, link, and much more.
-
-By adding metadata to their pools, pool owners can improve the discoverability of their pools on Balancer and make it easier for users to understand the purpose and characteristics of the pool.
-
-The Balancer Pool Metadata project provides an easy-to-use interface for pool owners to add and update their pool metadata, as well as a public Subgraph for developers to access and use the metadata.
+The tools are live at https://tools.balancer.bleu and https://balancer-tools.bleu.studio.
 
-### [`Internal Manager`](./packages/balancer-tools/src/app/internalmanager/)
+## Avaliable tools
 
-The Internal Manager is a user-friendly application that empowers individuals to take control of their internal balance by providing essential features to manage their assets efficiently.
+### Pool Metadata [🔗](https://tools.balancer.bleu/metadata)
```

Comment:
maybe because I'm at UFSC network kkkkk but I can not open the links

I get This site can’t be reached DNS_PROBE_FINISHED_NXDOMAIN

Reply:
fixed it. the suffix is `.blue` not `.bleu` 😬